src/solvespace.h

```diff
@@ -513,7 +513,7 @@ class SolveSpaceUI {
     GraphicsWindow              GW;
 
     // The state for undo/redo
-    typedef struct {
+    typedef struct UndoState {
```

Comment:
Yes, I've been meaning to make a pull request for this annoying warning.
But, again, in my opinion it does not belong in this his commit.